docs/fundamentals/networking/quic/quic-overview.md

```diff
@@ -265,7 +270,10 @@ await connection.CloseAsync(0x0C);
 await connection.DisposeAsync();
 ```
 
-or more information about how the `QuicConnection` was designed, see the [API proposal](https://github.com/dotnet/runtime/issues/68902).
+> [!NOTE]
```

Comment:
I wonder if we should move it under `QuicStream` section. We already have explanation within the example above.